src/Illuminate/Http/Middleware/PassThroughTrait.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Illuminate\Http\Middleware;
+
+trait PassThroughTrait
+{
+    /**
```

Comment:
@overtrue is right on this one, it just going to throw exception by defining the property in both place.